### DISCUSSION TITLE
[export/import] add ingredients to recipe-download

### DISCUSSION THREAD

**Discussion by lodriguez** [mealie-recipes/mealie#4430] [Feature Request]
### Please provide a concise description of the problem that would be addressed by this feature.

I've a local Mealie instance where I use 'Ingredient Amounts' for most recipes. Now, I want to share a recipe with someone. My plan was to simply hit the download button and send them the generated file so they can import it into their own Mealie instance.

However, there's an issue: this person doesn't have the same ingredients in their Mealie db. As a result, when they import the recipe, it will only show the amounts and comments, but the ingredients themselves will be missing.

### Please provide a concise description of the feature that would resolve your issue.

Include all ingredients in the zip file so they can be imported along with the recipe.

### Please consider and list out some caveats or tradeoffs made in your design decision

Well, importing a lot of ingredients could be inconvenient. An alternative solution could be to deactivate the 'Ingredient Amounts' option during export/download.

**Comment by wolsen9** [comment]
I would appreciate this.  For various reasons, I'd like to import only recipe data from a previous installation.  While exporting units, foods, etc. seem possible, there is no way to import them - this solution would achieve my intent!
